Vimeo-Native/VimeoInterop/VimeoInterop.vcxproj.filters

```diff
@@ -24,6 +24,9 @@
     <ClInclude Include="resource.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\Include\Vimeo\VimeoAPI.h">
```

Comment:
The file was already included in the main project file (VimeoInterop.vcxproj):
![image](https://cloud.githubusercontent.com/assets/5382895/9815979/1328e03c-5868-11e5-8dac-97ddb432f15a.png)
so I bet Visual Studio by default added it to the "Header Files" filter

Seems like it is probably ok